### PR TITLE
Searchsecomoney fixes

### PR DESCRIPTION
**Pull request by djeelun**
# Description

Fixed a the deployment to mumbai (replaced network with network.name). Also temporarily replaced token transfer with burning to test contract on mumbai. Finally added a temporary function to give tokens to an address to test hash paying.

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

File: contracts/facets/securesecoMonetization/SearchSECOMonetizationFacet.sol [hunk 1]
```diff
@@ -38,18 +40,25 @@ contract SearchSECOMonetizationFacet is AuthConsumer {
...
-        tokenContract.transferFrom(msg.sender, address(this), s.hashCost * _amount);
+        // tokenContract.transferFrom(msg.sender, address(this), s.hashCost * _amount);
+        IBurnableGovernanceStructure burnBridges = IBurnableGovernanceStructure(address(this));
+        burnBridges.burnVotingPower(msg.sender, s.hashCost * _amount);
         
         // Emit event so back-end can verify payment
         emit PaymentProcessed(msg.sender, _amount, _uniqueId);
     }
 
+    function giveMoneyz() external {
```
Comment (mik-0) [inline contracts/facets/securesecoMonetization/SearchSECOMonetizationFacet.sol:57]: Make name more professional

**Review by mik-0 — CHANGES_REQUESTED** [review]

File: contracts/facets/securesecoMonetization/SearchSECOMonetizationFacet.sol [hunk 2]
```diff
@@ -38,18 +40,25 @@ contract SearchSECOMonetizationFacet is AuthConsumer {
...
-        tokenContract.transferFrom(msg.sender, address(this), s.hashCost * _amount);
+        // tokenContract.transferFrom(msg.sender, address(this), s.hashCost * _amount);
+        IBurnableGovernanceStructure burnBridges = IBurnableGovernanceStructure(address(this));
+        burnBridges.burnVotingPower(msg.sender, s.hashCost * _amount);
         
         // Emit event so back-end can verify payment
         emit PaymentProcessed(msg.sender, _amount, _uniqueId);
     }
 
+    function mintMoneyToSender() external {
```
Comment (mik-0) [inline contracts/facets/securesecoMonetization/SearchSECOMonetizationFacet.sol:57]: Make name more accurate

**Review by mik-0 — CHANGES_REQUESTED** [review]

File: contracts/facets/securesecoMonetization/SearchSECOMonetizationFacet.sol (hunk 1 above)
Comment (djeelun) [reply contracts/facets/securesecoMonetization/SearchSECOMonetizationFacet.sol]: Sorry thanks for noticing! Updated it to mintTokensToSender

File: contracts/facets/securesecoMonetization/SearchSECOMonetizationFacet.sol (hunk 2 above)
Comment (djeelun) [reply contracts/facets/securesecoMonetization/SearchSECOMonetizationFacet.sol]: See reply to your previous request

**Comment by mik-0** [comment]
Bad PR, all temporary changes.

**Pull request closed without merging**